prospero-cli/src/test/resources/galleon-provisioning.xml

```diff
@@ -0,0 +1,30 @@
+<?xml version="1.0" ?>
+
+<!--
+  ~ Licensed to the Apache Software Foundation (ASF) under one or more
```

Comment:
can you change the license header to https://github.com/wildfly-extras/prospero/blob/main/prospero-common/src/main/java/org/wildfly/prospero/actions/Console.java#L1-L16 ?